FWCore/ParameterSet/src/types.cc

```diff
@@ -1115,3 +1272,38 @@ bool edm::encode(std::string& to, std::vector<ParameterSet> const& from) {
 }  // encode from vector<ParameterSet>
 
 // ----------------------------------------------------------------------
+
+#include <iostream>
```

Comment:
Is this `#include` a leftover from debugging?

Reply:
yep